src/data_cell/flatten_datacell.h

```diff
@@ -67,7 +67,10 @@ class FlattenDataCell : public FlattenInterface {
     }
 
     [[nodiscard]] const uint8_t*
-    GetCodesById(uint64_t id) const override;
+    GetCodesById(uint64_t id, bool& need_release) const override;
+
+    bool
+    GetCodesById(uint8_t* codes, uint64_t id) const override;
```

Comment:
Adjust the params sequence: Input before output

Reply:
done